qa/tasks/cephfs/mount.py

```diff
@@ -240,7 +240,7 @@ def _verify(self, proc, retval=None, errmsg=None):
             assert proc.returncode == retval, msg
 
         if errmsg:
-            stderr = proc.stderr.getvalue().lower()
+            stderr = proc.stderr.getvalue().decode().lower()
```

Comment:
```suggestion
            stderr = proc.stderr.getvalue().decode('utf-8').lower()
```

`decode` in python2 might default to something completely wrong.

Reply:
ah yeah, it is a unicode, still needs to use `six.ensure_str` here

Reply:
this part is still in flux. see #34386. i am inclined to hold back until these tests are settled.

Reply:
> ah yeah, it is a unicode, still needs to use `six.ensure_str` here

@kshtsk could you be more specific? what is a unicode. and why do we still need to use `six.ensure_str` here?

Reply:
@tchaikov I meant this:
```
>>> a=b'a'
>>> a.decode()
u'a'
```

Reply:
@tchaikov it is python2 specific class
```
Help on class unicode in module __builtin__:

class unicode(basestring)
 |  unicode(object='') -> unicode object
 |  unicode(string[, encoding[, errors]]) -> unicode object
```

Reply:
so *decode()* returns a _unicode_ which is not a standard _str_ which is actually _bytes_, latest python2 can transparently convert `str` to `unicode` in expressions where the latter occurs, which can make an unexpected results where it does matter.

Reply:
thanks @kshtsk and @sebastian-philipp . i am using `StringIO` to address this issue.